tests/fast-integration/installer/index.js

```diff
@@ -183,17 +183,45 @@ async function installRelease(
   debug(`Release ${release} deployed`);
 }
 
-async function chartList(options) {
-  const gardernerDomain = await getGardenerDomain();
-  const isGardener = process.env["GARDENER"] || (gardernerDomain) ? "true" : "false";
-  const domain = process.env["KYMA_DOMAIN"] || gardernerDomain || "local.kyma.dev";
+async function chartList(options) {  
+  const gardenerDomain = await getGardenerDomain();
+  const domain = process.env["KYMA_DOMAIN"] || gardenerDomain || "local.kyma.dev";
+
+  const isGardener = (gardenerDomain ? true : false) || (process.env["GARDENER"] === "true");
   const isCompassEnabled = !!options.withCompass;
-  const isCentralApplicationGatewayEnabled = !!options.withCentralApplicationGateway;
-  const overrides = `global.isLocalEnv=false,global.ingress.domainName=${domain},global.environment.gardener=${isGardener},global.domainName=${domain},global.tlsCrt=ZHVtbXkK,global.disableLegacyConnectivity=${isCompassEnabled},central_application_gateway.enabled=${isCentralApplicationGatewayEnabled},authProxy.config.useDex=false`;
+  const isCentralApplicationConnectivityEnabled = !!options.centralApplicationConnectivity;
+  const disableLegacyConnectivity = isCompassEnabled || isCentralApplicationConnectivityEnabled;
```

Comment:
Are you sure we want to disable LegacyConnectivity when the Central Application Connectivity is enabled? I think it may not be that tightly coupled

Reply:
That's right... My mistake